cynic-codegen/src/enum_derive/mod.rs

```diff
@@ -44,11 +45,22 @@ pub fn enum_derive_impl(
         None
     });
     if enum_def.is_none() {
+        let candidates = schema.definitions.iter().flat_map(|def| {
+            if let Definition::TypeDefinition(TypeDefinition::Enum(e)) = def {
+                Some(e.name.as_str())
+            } else {
+                None
+            }
+        });
+
+        let guess_field = guess_field(candidates, &(*(input.graphql_type)));
         return Err(syn::Error::new(
             input.graphql_type.span(),
             format!(
-                "Could not find an enum named {} in {}",
-                *input.graphql_type, *input.schema_path
+                "Could not find an enum named {} in {}.{}",
```

Comment:
Could we get an extra space in here please?

```suggestion
                "Could not find an enum named {} in {}. {}",
```